catboost/python-package/ut/medium/test.py

```diff
@@ -3664,3 +3664,34 @@ def test_eval_set_with_no_target_with_eval_metric(task_type):
     )
     with pytest.raises(CatboostError):
         model.fit(train_pool, eval_set=eval_set_pool)
+
+
+def test_model_comparison():
+    def fit_model(value):
```

Comment:
After changing to adult pool this function no longer depends on value so both model1 and model2 will be the same. I propose make 'iterations' a parameter and check 3 vs 5 iterations, for example.

Reply:
Done.